Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -187,7 +187,8 @@ bool GStreamerMediaEndpoint::initializePipeline()
         return false;
     }
 
-    if (gstObjectHasProperty(rtpBin.get(), "add-reference-timestamp-meta"))
+    auto disableCaptureTimeTracking = StringView::fromLatin1(g_getenv("WEBKIT_GST_DISABLE_WEBRTC_CAPTURE_TIME_TRACKING"));
+    if ((disableCaptureTimeTracking.isEmpty() || disableCaptureTimeTracking == "0"_s) && gstObjectHasProperty(rtpBin.get(), "add-reference-timestamp-meta"))
```

Comment:
I would first check the existance of the property and then if it exist, get the env var and do the value checks.